### PR TITLE
Community templates

### PR DESCRIPTION
**Pull request by onbjerg**
Adds two issue forms:

- Bug report
- Feature request

**You can see what the forms look like by going to "Files change" and clicking "View file" (accessible by the 3 dots in the top right) on the form**

Also adds:

- A link to the Foundry Support telegram when opening an issue
- A very minimal PR template
- Contributing guidelines (inspired by the one in ethers-rs)

Finally, does a pass on the README with some formatting nits and callouts to read the book.

For more info on the new issue forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

Anything missing in the forms?

### CODE REVIEW

**Comment by onbjerg** [comment]
Converted it to a draft because I also want to do a pass on the README + add a contributing doc

**Comment by onbjerg** [comment]
Ok, should be good now. Let me know!

**Review by mattsse — APPROVED** [review]
sick!
love the contributing guide, 
~~is this based on an existing one? if so we should reference that~~
ah adaptend from ethers-rs, gg

File: .github/ISSUE_TEMPLATE/BUG-FORM.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      description: What operating system are you on?
+      options:
+        - Windows
+        - macOS (amd)
+        - macOS (M1)
+        - Linux
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: Please include relevant Solidity snippets as well if relevant.
```
Comment (mattsse) [inline .github/ISSUE_TEMPLATE/BUG-FORM.yml:48]: should also add output of `forge --version`

Comment (onbjerg) [reply .github/ISSUE_TEMPLATE/BUG-FORM.yml:52]: Added! :smile:

File: README.md [hunk 2]
```diff
@@ -55,21 +60,21 @@ More documentation can be found in the [forge package](./forge/README.md) and in
...
-1. Portable (5-10MB) & easy to install statically linked binary without requiring Nix or any other package manager
-1. Abstracted over EVM implementations (currently supported: Sputnik, EvmOdin)
+- **Tests are written in Solidity** (like in DappTools)
+- **Fast fuzz testing** with shrinking of inputs & printing of counter-examples
+- **Fast remote RPC forking mode**, leveraging Rust's async infrastructure like tokio
+- **Flexible debug logging**
+    - Dapptools-style, using `DsTest`'s emitted logs
+    - Hardhat-style, using the popular `console.sol` contract
+- **Portable (5-10MB) & easy to install** without requiring Nix or any other package manager
+- **Abstracted over EVM implementations** (currently supported: Sputnik, EvmOdin)
```
Comment (onbjerg) [inline README.md:77]: Should we still list this? AFAIK this is no longer the case

Comment (mattsse) [reply README.md:77]: +1 on removing this, perhaps we can highlight the github action instead?

Comment (onbjerg) [reply README.md]: Done!

**Review by gakonst — APPROVED** [review]

**Pull request merged**